docs/fsharp/introduction-to-functional-programming/index.md

```diff
@@ -156,90 +156,15 @@ let addOneToValue x =
     x + 1
 ```
 
-Although this function does not depend on a global value, it writes the value of `x` to the output of the program. Although there is nothing inherently wrong with doing this, it does mean that the function is not pure.
+Although this function does not depend on a global value, it writes the value of `x` to the output of the program. Although there is nothing inherently wrong with doing this, it does mean that the function is not pure. If another part of your program depends on something such as the output buffer, then calling this function can affect that other part of your program.
```

Comment:
- "depends on something such as the output buffer" seems vague. Can there be some other dependency. Perhaps just "depends on the output buffer".